tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
The test setup slightly confuses me xD I would need a bit more time to adapt this. Feel free to fix it and open a PR against my branch. Otherwise geerlingguy could fix it.